src/Registry/TypeRegistry.php

```diff
@@ -1001,8 +1003,9 @@ public function setup_type_modifiers( $type ) {
 	 * @param array  $fields    Fields to register
 	 *
 	 * @return void
+	 * @throws \Exception
 	 */
-	public function register_fields( string $type_name, array $fields = [] ) {
+	public function register_fields( string $type_name, array $fields = [], ?bool $allow_underscores_in_field_names = false ): void {
```

Comment:
```suggestion
	public function register_fields( string $type_name, array $fields = [], bool $allow_underscores_in_field_names = false ): void {
```

I don't think we want to have to control for the possibility of `null`.

Reply:
correct. will update 🙏🏻

Reply:
ah, actually that arg needs removed. I changed the approach to happen in each field's `$config`. Removing that arg from the plural `register_fields()`